configs/disco/README.md

```diff
@@ -0,0 +1,149 @@
+# Disco Diffusion(Google Colab)
+
+> [](<>)
+
+> **Task**: Text2Image
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Disco Diffusion (DD) is a Google Colab Notebook which leverages an AI Image generating technique called CLIP-Guided Diffusion to allow you to create compelling and beautiful images from just text inputs. Created by Somnai, augmented by Gandamu, and building on the work of RiversHaveWings, nshepperd, and many others.
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/22982797/201001789-7ef108a0-f607-401e-98dc-4e16d6be384f.png"/>
+</div>
+
+## Models Card
+
+## Quick Start
+
+In order to get started, we introduce a simplest way to get an image within 6 line of codes.
+
+```python
+from mmengine import Config, MODELS
+from mmedit.utils import register_all_modules
+register_all_modules()
+
+disco = MODELS.build(Config.fromfile('configs/disco/disco-baseline.py').model).cuda().eval()
+text_prompts = {
+    0: ["A beautiful painting of a singular lighthouse, shining its light across a tumultuous sea of blood by greg rutkowski and thomas kinkade, Trending on artstation.", "yellow color scheme"]
+}
+image = disco.infer(height=768, width=1280, text_prompts=text_prompts, show_progress=True, num_inference_steps=250, eta=0.8)['samples']
+```
+
+## Advanced Tutorials
+
+For detailed description and advanced usage.
+
+### Overall Architecture(In Construction)
+
+### Infer Settings
+
+For fixed Disco-Diffusions, there are several runtime settings.
```

Comment:
what does 'fixed Disco-Diffusions' mean?

Reply:
For a defined `DiscoDiffusion` Model